tools/codemod/src/utils/TransformStagePanel.ts

```diff
@@ -0,0 +1,207 @@
+import { JSCodeshift, ASTPath, JSXElement, ObjectProperty, SpreadProperty, Property, ObjectExpression } from "jscodeshift";
+import { concatExpressions } from "./ExpressionHandles";
+import { buildProperty, ConfigExpression, ExpressionKind, getObjectProperty, isExpressionKind, jsxElementToUniFormat } from "./TransformUtils"
+
+export function transformStagePanel(j: JSCodeshift, stagePanel: ASTPath<JSXElement>): ConfigExpression { // TODO: registerMethod
```

Comment:
Functions that are v4.0.0 specific, shouldn't live under `utils`.

Reply:
Also, I will rename `frontstage-to-config` to `element-to-config` if that's alright

Reply:
Reusable part should be in the `utils` directory, transformer specific part should be contained in the transfomer file itself.
Sure for the rename.